_shows/16_17/monarch.md

```diff
@@ -39,6 +39,16 @@ assets:
   - type: poster
     image: 58FMrT8
 
+links:
+  - type: Review
+    href: http://www.impactnottingham.com/2017/03/monarch-nnt/
+    snapshot: EvgYq
+    publisher: Impact Nottingham
+    author: Ashleigh John
+    title: "Monarch @ Nottingham New Theatre"
+    date: 2017-03-29
+    rating: 9/10
+    quote: " All of the actors should be highly commended for their outstanding acting; each contributing to making this play, which deals with heavy themes such as political radicalism, rape and disconnection, heart-wrenching yet relatable for the audience. "
```

Comment:
Extra spaces at start and end of these strings. (Also in other docs). Can we clean this up?